designer-demo/src/composable/http/mock.js

```
@@ -0,0 +1,939 @@
+        return response.json()
+      }
+
+      const blocks = await getAvailable()
+      const tags = []
+
+      blocks &&
+        blocks.forEach((block) => {
+          block.tags && tags.push(...block.tags)
+        })
+
+      return new Promise((resolve) => {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)